internal/e2e/evidence_query_test.go

```diff
@@ -317,8 +317,12 @@ var _ = Describe("Updating evidence via API", Label("e2e", "Evidences"), func()
 				evidence.Description = "New Description"
 
 				req.Var("id", fmt.Sprintf("%d", evidence.Id))
-				req.Var("input", map[string]string{
+				req.Var("input", map[string]interface{}{
 					"description": evidence.Description,
+					"severity": model.SeverityInput{
```

Comment:
Why is this needed in the evidence test?

Reply:
I removed this now as it's not needed and works without the severity input now.